mgmt/rest/log_handler.go

```diff
@@ -50,6 +52,17 @@ func (l *Logger) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.Ha
 		"method":      r.Method,
 		"url":         r.URL.Path,
 		"status-code": res.Status(),
-		"status":      http.StatusText(res.Status()),
+		"status":      deprecated(http.StatusText(res.Status()), rw.Header().Get("Deprecated"), rw.Header().Get("DeprWhat"), rw.Header().Get("DeprInfo")),
 	}).Debug("API response")
 }
+
+// 'deprecate' is string boolean value to deprecate, mapped to 'Deprecated' key
+// 'deprWhat' is what is being deprecated, mapped to 'DeprWhat' key
+// 'deprInfo' is link to more information on what is being deprecated, mapped to 'DeprInfo' key
+func deprecated(status string, deprecate string, deprWhat string, deprInfo string) string {
+	dep, err := strconv.ParseBool(deprecate)
+	if err != nil || !dep {
+		return status
+	}
+	return status + ". '" + deprWhat + "' has been depricated. Find more information here: " + deprInfo
+}
```

Comment:
I have a doubt if this is really need to be done in this way. What do you think about making it more straightforward, e.g.: 
```
       if deprecationInfo := rw.Header().Get("Deprecated"); len(deprecationInfo) != 0 {
                restLogger.WithFields(log.Fields{
                        "method": r.Method,
                        "url":    r.URL.Path,
                }).Warning(deprecationInfo)
        }
```

where in [mgmt/rest/v1/rbody/body.go](https://github.com/intelsdi-x/snap/pull/1638/files#diff-d53afab335763da0de3e3036f60107faR43) const might determined the string value which is set in Write():
```
const deprecationInfo = "API V1 has been depricated. Find more information here: https://github.com/intelsdi-x/snap/issues/1637"

func Write(code int, b Body, w http.ResponseWriter) {
        w.Header().Set("Deprecated", deprecationInfo)
[...]
}
```